apps/api/src/domains/admin/user/user.service.ts

```diff
@@ -141,15 +138,6 @@ export class UserService {
     await this.userRepo.remove(users);
   }
 
-  async validateEmail(email: string) {
-    const tenant = await this.tenantService.findOne();
-    const domain = email.split('@')[1];
-    if (tenant.isRestrictDomain && !tenant.allowDomains?.includes(domain)) {
-      throw new NotAllowedDomainException();
-    }
-    return true;
-  }
-
```

Comment:
Why should this function be moved to member service?
As email is a property for user, I think it would be more natural in user service.
Is there any reason for this move?

Reply:
This is due to the circular dependency. 
It's quite complicated to explain in detail, but I could workaround by disconnecting the relationship between user service and tenant service. 
Also, that's the reason why I use `tenantRepo.find()` rather than `tenantService.findOne()`